systemFlake.nix

```
@@ -40,7 +40,8 @@ let
     , builder ? channels.${channelName}.input.lib.nixosSystem
     , modules ? [ ]
     , extraArgs ? { }
-    }: { inherit channelName system output builder modules extraArgs; };
+    , specialArgs ? { }
```

```suggestion
    # These are not part of the module system, so they can be used in `imports` lines without infinite recursion
    , specialArgs ? { }
```
Just to explain the difference, so others don't have to hunt down comments in lib/modules.nix